examples/example-app/README.md

```diff
@@ -0,0 +1,15 @@
+# Example of iland Cloud Java SDK
+In this example app, we use the Java SDK to perform various operations.<br />
+With the SDK, we login into the console. Then we get and print all the permissions for the user.
```

Comment:
permissions?

Reply:
I also wouldn't say that we log in to the console because this isn't related to the console in any way